### PR TITLE
Prevent duplicate Domains to be created using the API

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

File: webapps/api/backend/views/domains.py [hunk 1]
```diff
@@ -72,6 +72,10 @@ def create(self, request, contents=None):
         contents = None if contents is None else contents.split(',')
         serializer = FullSerializer(Domain, contents=contents, instance=Domain(), data=request.DATA)
         domain = serializer.deserialize()
+        current_domains = DomainList.get_by_name(domain.name)
+        if len(current_domains) > 0:
+            raise HttpNotAcceptableException(error_description='A Domain with the given name already exist',
```
Comment (kvanhijf) [inline webapps/api/backend/views/domains.py]: `exists`

File: ovs/dal/migration/ovsmigrator.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ class OVSMigrator(object):
     """
 
     identifier = 'ovs'
-    THIS_VERSION = 14
+    THIS_VERSION = 15
```
Comment (kvanhijf) [inline ovs/dal/migration/ovsmigrator.py]: Why bump this when no migration code is present?

**Review by kvanhijf — CHANGES_REQUESTED** [review]

File: ovs/dal/migration/ovsmigrator.py (hunk 2 above)
Comment (khenderick) [reply ovs/dal/migration/ovsmigrator.py]: Added an index, so the migration code for indexes has to run again.

Comment (kvanhijf) [reply ovs/dal/migration/ovsmigrator.py]: Yes, but since we're still in development, i guess this isn't required?

Comment (khenderick) [reply ovs/dal/migration/ovsmigrator.py]: Hmm, valid point. I'll keep it on 14

**Review by kvanhijf — APPROVED** [review]

**Pull request merged**